subtitles.go

```diff
@@ -266,27 +266,40 @@ func (sa *StyleAttributes) propagateTeletextAttributes() {
 	}
 }
 
+//reference for migration: https://w3c.github.io/ttml-webvtt-mapping/
 func (sa *StyleAttributes) propagateTTMLAttributes() {
 	if sa.TTMLTextAlign != nil {
 		sa.WebVTTAlign = *sa.TTMLTextAlign
 	}
-}
-
-func (sa *StyleAttributes) propagateTTMLRegionAttributes() {
 	if sa.TTMLExtent != nil {
+		//region settings
 		lineHeight := 5 //assuming height of line as 5.33vh
 		dimensions := strings.Split(*sa.TTMLExtent, " ")
```

Comment:
Can you wrap below logic inside `if len(dimensions) > 1`

Indeed if `len(dimensions) == 1`, using `dimensions[1]` will panic

Reply:
Done